azure-container-apps/ADVANCED.md

```diff
@@ -0,0 +1,321 @@
+```
+
+### Container App Name requirements
+
+Your Container App Name (the `ConAppName` variable) can contain lowercase letters, numbers, and hyphens. It must be 2 to 32 characters long, cannot start or end with a hyphen, and cannot start with a number. [Learn more about the naming rules and restrictions for Azure resources](https://learn.microsoft.com/azure/azure-resource-manager/management/resource-name-rules#microsoftapp).
+
+### Viewing logs in Azure Container Apps
+
+You can [view logs for a container app](https://learn.microsoft.com/azure/container-apps/log-streaming?tabs=bash#view-log-streams-via-the-azure-portal) in the **Log Stream** area of your environment or container app in the Azure portal. If you're having an issue starting up the SCIM bridge, reviewing the **op-scim-bridge** container logs can help you identify the problem.
+
+### How to update the **scimsession** secret
+
+After you download a new `scimsession` file, follow the steps below to replace the secret in your Container App.
+
+Replace your <code>scimsession</code> secret using the Azure Cloud Shell or AZ CLI
+
+> The following steps assume you have moved to mounting your secret from a volume mount and not using the base64 value in your secrets. 
+> Follow the [command to update your deployment with the updated YAML file](https://support.1password.com/scim-deploy-azure/#step-3-set-up-and-deploy-1password-scim-bridge) to use volume mounts, redefining your variables as needed for this command to succeed. 
+
+
+1. Open the [Azure Shell](https://shell.azure.com) or use the `az` CLI tool.
+
+2. Copy and paste the following command, replace `$ConAppName` and `$ResourceGroup` with the names from your deployment, and run the command.
+
+    - **Bash**:
+
+        ```bash
+        az containerapp secret set \
+            --name $ConAppName \
+            --resource-group $ResourceGroup \
+            --secrets scimsession="$(cat $HOME/scimsession)"
+        ```
+
+    - **PowerShell**:
+
+        ```pwsh
+        az containerapp secret set `
+            --name $ConAppName `
+            --resource-group $ResourceGroup `
+            --secrets scimsession="$(Get-Content $HOME/scimsession)"
+        ```
+
+3. Copy and paste the following command, which will have the `op-scim-bridge` container read the new secret. Replace `$ConAppName` and `$ResourceGroup` with the names from your deployment, then run the command.
+    ```bash
+    az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge --query properties.latestRevisionName
+    ```
+
+    Update the revsion name to use the output of the above command
+    ```
+    az containerapp revision restart -n $ConAppName -g $ResourceGroup --revision revisionName
+    ```
+
+4. Open your SCIM bridge URL in a browser and enter your bearer token to test the bridge.
+
+5. Update your identity provider configuration with the new bearer token.
+
+### Updating your SCIM bridge in the Azure Cloud Shell
```

Comment:
As mentioned above, this should be an H2 and it should be above the troubleshooting section.

Reply:
Moved Update your SCIM bridge section above Get Help section in latest commit.